lib/gringotts/gateways/monei.ex

```diff
@@ -377,10 +377,11 @@ defmodule Gringotts.Gateways.Monei do
                    "authentication.entityId": opts[:entityId]]
     body = params ++ auth_params
     url = "#{base_url(opts)}/#{version(opts)}/#{endpoint}"
-    case method do
-      :post -> HTTPoison.post(url, {:form, body}, @default_headers) |> respond
-      :delete -> HTTPoison.delete(url  <> "?" <> URI.encode_query(auth_params)) |> respond
-    end
+    network_response = case method do
+                         :post -> HTTPoison.post(url, {:form, body}, @default_headers)
```

Comment:
@oyeb we discussed formatting for this as per latest formatter code, make that change.